components/feature/awesomebar/src/main/java/mozilla/components/feature/awesomebar/provider/BookmarksStorageSuggestionProvider.kt

```diff
@@ -71,7 +71,6 @@ class BookmarksStorageSuggestionProvider(
                 flags = setOf(AwesomeBar.Suggestion.Flag.BOOKMARK),
                 title = result.title,
                 description = result.url,
-                editSuggestion = result.url,
```

Comment:
I don't think it's right to remove this for the `Awesomebar` in general. Maybe you want to be able to configure this so it can behave different for bookmark and history search?

Reply:
One easy option is to add a parameter to the provider to configure whether or not to add edit suggestions.

Reply:
Sure, I'll make it configurable so we can turn it on or off.  Thanks